src/__tests__/cssls.test.ts

```diff
@@ -0,0 +1,172 @@
+import { Matrix } from 'ml-matrix';
+
+import { cssls } from '../cssls';
+import initialisation from '../initialisation';
+
+let result: Matrix;
+let solution: Matrix;
+afterEach(() => {
+  for (let i = 0; i < result.rows; i++) {
+    for (let j = 0; j < result.columns; j++) {
+      expect(result.get(i, j)).toBeCloseTo(solution.get(i, j), 2);
+    }
+  }
+});
```

Comment:
This isn't how you should do it. Write a function that does the assertion and call it from the tests.

Instead of

```ts
solution = x
result = y
// assume something is tested afterEach
```

```ts
assertResult(result, solution);
```